content/docs/18-knowledgebase/01-tutorials/10-dev-engine/03-deploy-k8s-cluster.md

```diff
@@ -0,0 +1,1711 @@
+
+<br />
+
+# Verify the Application
+
+From the cluster details page, click on **Workloads** at the top of the page:
+
+![Workloads](/tutorials/deploy-clusters/workloads.png)
+
+The tab opens an overview of the Kubernetes components. From there, you can check if the application components have been created successfully.
+
+Select the **Namespaces** tab and check for a namespace called *cluster-xxxxxx*.
+
+Select the **Deployments** tab and check the existence of a deployment with name *hello-universe-deployment*. In the **Deployments** tab you can verify the status of the deployment: next to the deployment name, check the number of Pods ready and the number of replicas to know if the application is fully deployed.
+
+Select the **Pods** tab and check for two pods with name *hello-universe-deployment-xxxxxx*. In the **Pods** tab, next to the pods names, check the status of the pods
+
+![Pods status](/tutorials/deploy-clusters/deploy_app/app_update_pods.png)
+
+Remember to update the list of components by clicking the *sync* button at top right corner to refresh and synchronize the current state of the Kubernetes components.
+
+<br />
+
+To access the application, select the **Overview** tab at the top of the page and click on the link (**:8080**) that appears next to the *hello-universe-service* in the **Services** line. It will open a new tab on the browser that open the application.
+
+![Deployed application](/tutorials/deploy-clusters/app.png)
+
+<br />
+
+
+# Clean-up
+
+It is a good practice to clean up the resources used for the tutorial from the cloud provider, to avoid unexpected cloud charges.
+
+<br />
+
+<Tabs>
+<Tabs.TabPane tab="UI Workflow" key="ui-clean">
+
+## UI Workflow
+
+To remove the cluster created in this tutorial, open the Palette dashboard and, from the left **Main Menu** click on the **Cluster** panel to access the clusters page. Select the cluster you want to delete to access its details page.
+
+Click on **Settings**, at the top-right corner of the page, from the details page to expand the settings menu and select **Delete Cluster** to delete the cluster.
+
+![Destroy-cluster](/tutorials/deploy-clusters/delete-cluster-button.png)
+
+You will be asked to type in the cluster name to confirm the delete action. Continue and type the cluster name to proceed with the delete step. Repeat this process for all the cluster you want to delete.
+
+</Tabs.TabPane>
+<Tabs.TabPane tab="Terraform" key="terraform-clean">
+
+## Terraform
+
+To destroy the cluster with Terraform, use the destroy command.
+
+First enter into the folder where you have the Terraform configuration for the cluster
+```bash
+$ cd terraform-cluster
+```
+
+If you want to check the resources you will delete, you can first execute: 
+```terraform
+terraform plan -destroy
+```
+
+Then delete the components by running the destroy command:
+```terraform
+$ terraform destroy
+```
+
+Wait until it finishes to delete the cluster.
```

Comment:
```suggestion
The deletion process will take a few minutes. 
```
Let's also include the output that shows the count of resources removed and let's add a note about force delete. 

![CleanShot 2023-04-07 at 10 09 48](https://user-images.githubusercontent.com/29551334/230649185-1dca3de1-7670-40cc-b2b9-6c69665fae14.png)